crates/cli-support/src/js/mod.rs

```diff
@@ -460,6 +461,37 @@ impl<'a> Context<'a> {
 
         self.unexport_unused_internal_exports();
         closures::rewrite(self)?;
+
+        // Handle the `start` function, if one was specified. If we're in a
+        // --test mode (such as wasm-bindgen-test-runner) then we skip this
+        // entirely. Otherwise we want to first add a start function to the
+        // `start` section if one is specified.
+        //
+        // Afterwards, we need to perfom what's a bit of a hack. Right after we
```

Comment:
Nitpick: "perfom" -> "perform"

```suggestion
        // Afterwards, we need to perform what's a bit of a hack. Right after we
```